components/CustomBranding.tsx

```diff
@@ -0,0 +1,10 @@
+import { useEffect } from "react";
+
+const BrandColor = ({ val }: { val: string }) => {
```

Comment:
```suggestion
const BrandColor = ({ val = "#292929" }: { val: string }) => {
```

So we don't have to do `<CustomBranding val={props.profile.brandColor || "#292929"} />` every single time.

Just keep it like `<CustomBranding val={props.profile.brandColor} />`